releasenotes/notes/inventories_network_use_hostname_resolver-1c19f97cd2208277.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Adds a config option to use hostname DNS resolution as the
+    preferred method to compute the host IP address
```

Comment:
We should mention the setting name here and we should update the config template if this is a publicly documented setting.

Reply:
I'm not 100% sure we want to make this publicly available yet, we definitely don't have enough customer data regarding the need. 

But agreed on adding the setting name in the release note, I'll update

Reply:
Done in [address review](https://github.com/DataDog/datadog-agent/pull/35145/commits/9445dd6ad2f9317c46b0b5f134d517ca9ccb19a4)

Reply:
We can always add the feature without a release note :shrug: